src/renderer/components/+preferences/application.tsx

```diff
@@ -50,6 +52,7 @@ export const Application = observer(() => {
     );
 
   const [shell, setShell] = React.useState(UserStore.getInstance().shell || "");
+  const extensionSettings = AppPreferenceRegistry.getInstance().getItems().filter((preference) => preference.showInPreferencesTab === "application");
```

Comment:
Maybe we could just move all preference pages to this style?

Reply:
do you mean add settings from extensions to other preference pages too?

Reply:
Yes, I don't quite see why not.

Reply:
I don't know. This is meant for bundled extensions? This PR puts application extensions  after the "other" section and before "update-channel", to suit the mockup. Where do we place extension preferences on other pages? It would be better if the placements were configurable, but that's another ballgame. Otherwise implement support on other pages if the need arises?

Reply:
We don't even have support for configuring the order of preferences on the extensions page so I don't think we have to solve that problem too.

Though if we were to do that then we probably should just follow the lead of all other orderings within Lens and use "priority"

Reply:
For sure, I meant that I don't see the point of adding extension settings to the other pages because they don't have such ordering specifications, and without that it is all ad hoc. Add them if and when we need them to match some other mockup.

Reply:
I agree with @jim-docker. If there will be more of these requirements then we'd like to have some ordering in place. Maybe follow YAGNI for now?